camayoc/tests/qpc/cli/test_openshift.py

```diff
@@ -77,6 +78,50 @@ def validate_openshift_report(cluster, details, deployments):
     assert set(cluster_nodes) == set(seen_nodes), "The number of expected nodes diverged"
 
 
+def validate_node_metrics(cluster, details):
+    node_metrics = details["sources"][0]["facts"][-1]["node_metrics"]
+    for metric in node_metrics:
+        package = ast.literal_eval(metric["package"])
```

Comment:
Do we need `literal_eval`, which comes with a number of warnings in documentation? Can't we just cast this to `int`? I assume this is a number based on assertion in the next line.